lib/SILOptimizer/Mandatory/AddressLowering.cpp

```diff
@@ -2978,6 +2971,14 @@ class DefRewriter : SILInstructionVisitor<DefRewriter> {
     for (Operand &operand : tupleInst->getAllOperands())
       addrMat.initializeComposingUse(&operand);
   }
+
+  void markRewritten(SILValue oldValue, SILValue addr) {
```

Comment:
This doesn't actually mark the storage rewritten. So maybe it should be called something else. Like `setStorageAddress`